code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -59,10 +59,12 @@
 					if(cpr_cooldown < world.time)
 						revive_grace_period += 7 SECONDS
 						attacking_mob.visible_message(SPAN_NOTICE("<b>[attacking_mob]</b> performs <b>CPR</b> on <b>[src]</b>."),
-							SPAN_HELPFUL("You perform <b>CPR</b> on <b>[src]</b>."))
+							SPAN_HELPFUL("You perform <b>CPR</b> on <b>[src]</b>."),
+							balloon_alert(usr, "you perform cpr with success"))
```

Comment:
The only thing I can think of in regard to the text would be removing the "with success" part from this one, since it's sort of implied already.
```suggestion
						balloon_alert(attacking_mob, "you perform cpr")
```